crates/biome_service/src/configuration/linter/mod.rs

```diff
@@ -175,20 +172,37 @@ impl Merge for RuleConfiguration {
     }
 }
 
-impl From<&RuleConfiguration> for Severity {
-    fn from(conf: &RuleConfiguration) -> Self {
+impl<T: Clone + Default + 'static> RuleConfiguration<T> {
```

Comment:
Why is the `'static` required here?

Reply:
It is required because of `RuleOptions::new<O: 'static>`. This is in turn required because we use `TypeId::of::<O>`.